Parcial de preogramación 1/israel boksh parcial #1.cpp

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+ main ()
+{
+	
+	if ("num d.50 m" .50) ( num e"100*d");
+	else ("% d" d.50* 4);
+	
+	Printf("\n multiplicar la parte entere\n;");
+	break;
+	
+	if(4 d*a=5 , d* a=11);
+	else("% d",num*4);
+	
+	printf("\n no es divisible\n,");
+	break;
+	
+	it(" numd.50 m" .50*4);

```

Comment:
Esta funcion it, no existe

Reply:
![images](https://user-images.githubusercontent.com/31838625/30382672-62df3c2c-9865-11e7-8dad-180a647cddf0.jpg)